.isort.cfg

```diff
@@ -1,16 +1,6 @@
 [settings]
-include_trailing_comma=True
-force_grid_wrap=0
-use_parentheses=True
+profile=black
```

Comment:
New isort version has a profile for compat with the black autoformatter, so switching to that.